_studio/mfx_lib/encode_hw/shared/ehw_device_vaapi.cpp

```diff
@@ -339,16 +339,20 @@ mfxStatus DeviceVAAPI::Init(
 mfxStatus DeviceVAAPI::QueryStatus(DDIFeedback& ddiFB, mfxU32 id)
 {
     MFX_CHECK(!ddiFB.Get(id), MFX_ERR_NONE);
+    auto itMB = std::find_if(ddiFB.ExecParam.begin(), ddiFB.ExecParam.end(), DDIExecParam::IsFunction<VAFID_MapBuffer>);
```

Comment:
which encoders are affected by this common code? is vp9 touched?

Reply:
HEVC is affected here

Reply:
ok, thx.